### PR TITLE
[actions] Add GitHub action to compute the changelog for maestro bumps.

### PR DESCRIPTION
**Pull request by rolfbjarne**
And add that changelog as a comment to the PR. If there already is a comment
with a changelog, then update that comment instead.

### CODE REVIEW

File: .github/workflows/maestro-changelog.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    name: Add changelog
+    runs-on: ubuntu-latest
+    if: github.actor == 'dotnet-maestro[bot]'
+
+    steps:
+    - name: 'Dump environment'
+      run: |
+        set -e
+        set -x
+        env | sort
```
Comment (chamons) [inline .github/workflows/maestro-changelog.yml:15]: Is this desired or left over debugging aid?

**Review by chamons — APPROVED** [review]
Seems reasonable, but I'm curious about the usage of github-script. FWICT we've mostly standardized on powershell for CI scripting. Was this a pre-baked script we're leveraging or something new?

**Review by dalexsoto — APPROVED** [review]
Nice!

File: .github/workflows/maestro-changelog.yml (hunk 1 above)
Comment (rolfbjarne) [reply .github/workflows/maestro-changelog.yml]: Removed!

**Pull request merged**